docs/source/user/fast.farm/FFarmTheory.rst

```diff
@@ -1291,6 +1292,135 @@ the solution of the momentum equation),
 :math:`V_{r_{n_p,n_r-1}}^\text{Wake}\left[ n+1 \right]` for
 :math:`1\le n_r\le N_r-1`. [10]_
 
+
+
+.. _FF:WAT:
+
+Wake-Added Turbulence (WAT)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Wake-added turbulence is the additional small-scale turbulence generated
+from the turbulent mixing in the wake. 
+It is modeled by scaling up a background (undisturbed) turbulence.
+
+
+The theory for WAT will be presented in more details in the future as part of a journal publication.
```

Comment:
We can address this later, but I would say that the journal paper should document the WAT model at high-level, and the details regarding the implementation in FAST.Farm should be stated here.  (This would be consistent with the rest of the FAST.Farm theory section documented here.)